### DISCUSSION TITLE
Which version of `sbt` are currently supported?

### DISCUSSION THREAD

**Discussion by kraktus** [sbt/sbt#6607] [Q&A] [answered]
I could not find this information on the website.

Are the branches of this repo an official stance on that matter?

Related to https://github.com/Homebrew/homebrew-core/pull/81866

**Comment by eed3si9n** [comment, marked as answer]
For open source, sbt project normally supports only the latest patch version of the latest stable branch -- of as of July, 2021 that would be sbt 1.5.5. In this case, I mean support to roughly mean that we'll try to address bug reports etc.

When you backquote as `sbt`, that could mean `sbt`-the-Bash-script (also called the runner or launcher), installed by different distribution mechanism such as Home Brew. I actively avoid using Home Brew and discourage anyone from using it due to https://github.com/Homebrew/homebrew-core/issues/50649. Some people may use Home Brew to install `sbt`, but they should know that they do so at their own risk.

Depending on what you mean by _supported_, there could be more or less information, so I'm not sure if this answers the question. For example, most `sbt`-the-Bash-script should _support_ the execution of sbt versions based on `project/build.properties`.

With regard to Apple Silicon (M1, Arm Mac etc), we have made some experimental patches to support it, but it probably should be considered somewhat experimental. See also
- https://github.com/sbt/sbt/issues/6162
- https://github.com/sbt/io/pull/321

**Reply by kraktus** [reply]
Thank you for the detail answer! Sorry for the delay
